libethcore/TransactionBase.cpp

```diff
@@ -64,23 +64,25 @@ TransactionBase::TransactionBase(bytesConstRef _rlpData, CheckTransaction _check
 
 		m_data = rlp[5].toBytes();
 
-		byte v = rlp[6].toInt<byte>();
-		h256 r = rlp[7].toInt<u256>();
-		h256 s = rlp[8].toInt<u256>();
+		u256 const v = rlp[6].toInt<u256>();
```

Comment:
If we use `int` later on, can we use `toInt<int>()` here?

Reply:
Yes, I think it makes sense

That way we'll fail when parsing RLP if it's bigger than `int` instead of silently overflowing